src/transformers/sinusoidal_positional_embeddings.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import torch
+from torch import nn
+
+
+class SinusoidalPositionalEmbedding(nn.Embedding):
```

Comment:
Do we need an extra file for this or could we leave it in Bart for the moment?

Reply:
I think there are multiple models which use SinusoidalPositionalEmbedding (distilbert, xlnet, ...) and they are all in their respective files. Just for consistency and it seems like too small to have its own file for me.

Reply:
Done, makes sense.